### ISSUE TITLE
EPIC: Model Training, Optimization and Validation

### ISSUE THREAD

**Issue by Hasmelash95** [Hasmelash95/breast-cancer-detector#3]
Train the model using the data in the training set and use the validation set for optimum accuracy of the prediction.
